app/migrations/seed.py

```diff
@@ -171,6 +168,44 @@ def create_ussd_menus():
         parent_id=account_management_menu.id
     )
 
+    print('********* User Profile Menu **********')
+    update_or_create_menu(
+        name='first_name_entry',
+        description='First name entry prompt.',
+        parent_id=user_profile_menu.id
+    )
+
+    update_or_create_menu(
+        name='last_name_entry',
+        description='Last name entry prompt.',
+        parent_id=user_profile_menu.id
+    )
+
+    update_or_create_menu(
+        name='gender_entry',
+        description='Gender entry prompt.',
+        parent_id=user_profile_menu.id
+    )
+
+    update_or_create_menu(
+        name='location_entry',
+        description='Location entry prompt.',
+        parent_id=user_profile_menu.id
+    )
+
+    update_or_create_menu(
+        name='change_my_business_prompt',
+        description='Change business directory info',
+        parent_id=user_profile_menu.id
+    )
+
+    update_or_create_menu(
```

Comment:
I propose remove from create, only have on update. I hate verbose signups myself.

Reply:
I am thinking the method is appropriately named because in the event that there is an existing menu, it will update and if not, it will create ??

Reply:
ah, it's creating the menu - its not the menu for creating and updating?